assets/global.js

```diff
@@ -598,25 +613,16 @@ class VariantSelects extends HTMLElement {
   updateMedia() {
     if (!this.currentVariant) return;
     if (!this.currentVariant.featured_media) return;
-    const newMedia = document.querySelector(
-      `[data-media-id="${this.dataset.section}-${this.currentVariant.featured_media.id}"]`
-    );
 
-    if (!newMedia) return;
+    this.stickyHeader = this.stickyHeader || document.querySelector('sticky-header');
+    if (this.stickyHeader) this.stickyHeader.dispatchEvent(new Event('preventHeaderReveal'));
```

Comment:
I think this might need to be placed elsewhere 🤔 
[video](https://screenshot.click/21-11-6z3mn-ijgto.mp4)

Also when the sticky header is showing and you click on a variant image, it should probably hide the sticky header if it's currently sticky I think. You can see it at 16s in the video above.

Reply:
Moved to the MediaGallery component! :)